crates/napi/src/next_api/project.rs

```diff
@@ -1266,8 +1266,8 @@ pub async fn project_trace_source(
                     Some(source_file.to_string()),
                     false,
                 )
-            } else if let Some(source_file) = original_file.strip_prefix(SOURCE_MAP_PREFIX) {
-                // All other code like turbopack://[turbopack] is internal code
+            } else if let Some(source_file) = original_file.strip_prefix(SOURCE_URL_PROTOCOL) {
```

Comment:
The three slashes are missing here, intentional?

Reply:
No but I also don't understand this branch. Isn't this saying that we resolve a `turbopack:///[turbopack]/file.ts` to `///[turbopack]/file.ts` (or previously `[turbopack]/file.ts`. That seems like it's not a desired location since it's no longer clear what `[turbopack]` stands for.